pandas/tests/indexes/test_numeric.py

```diff
@@ -919,6 +919,10 @@ def test_constructor(self):
         res = Index([1, 2**63])
         tm.assert_index_equal(res, idx)
 
+        idx = Index([-1, 2**63], dtype=object)
```

Comment:
add test when dtype is specified (and it fails)
e.g. uint64 with neg and also with nan